Shipfinity.DataAccess/Repositories/Interfaces/IProductRepository.cs

```diff
@@ -11,6 +11,12 @@ public interface IProductRepository : IRepository<Product>
     {
         Task<List<Product>> GetAllByCategoryIdAsync(int categoryId);
         Task<List<Product>> GetRangeOrderedByPrice(int price, int start, int count);
+        Task UpdateProductPhotoUrlAsync(int productId, string photoUrl);
+
```

Comment:
Remove unnecessary padded new lines, because there may cause further conflicts in the future